chart/fluentbit-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{  .Values.metadata.name  }}
+  namespace: {{  .Values.metadata.namespace  }}
+  labels:
```

Comment:
The `label`  is a fixed value and does not need to be set as configurable.

Similarly, `clusterrole`, `clusterrolebinding`, `serviceacount` should all be fixed values